hack/tools.go

```diff
@@ -24,6 +24,10 @@ import (
 	_ "github.com/gardener/gardener/hack"
 	_ "github.com/gardener/gardener/hack/.ci"
 	_ "github.com/gardener/gardener/hack/api-reference/template"
+
+	// TODO(KA): This is imported to suppress the warning caused by the gardener/gardener tools. It can be
```

Comment:
You can simply drop 408d27a instead

Reply:
But then the warning will be printed. I think about it like so: if we already had to scratch our head then until we vendor the next gardener release, how many contributors will face the same ? I can live with an extra (unused) file in the repo instead.